### ISSUE TITLE
Snakemake +Singularity with "oras://" URLs

### ISSUE THREAD

**Issue by chrarnold** [snakemake/snakemake#1263]
Is Snakemake already supporting the "oras" (see the Singularity changelog [here](https://github.com/hpcng/singularity/blob/master/CHANGELOG.md#v330---2019-06-17) registries? Before I create them, would be good to have it confirmed this works flawlessly, thank you! I ask because there may be something hard-coded so that "oras" URLs wont work, our Singularity at work supports it.

**Comment by boulund** [comment]
Yes, it works just fine to have rules with `oras://` URLs in the `singularity:` directive.
